wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java

```diff
@@ -763,6 +764,8 @@ public void onCreate() {
         application.getWallet().addCoinsSentEventListener(Threading.SAME_THREAD, walletEventListener);
         application.getWallet().addChangeEventListener(Threading.SAME_THREAD, walletEventListener);
 
+        application.getWallet().getContext().sporkManager.addEventListener(sporkUpdatedEventListener, Threading.SAME_THREAD);
+
```

Comment:
The `sporkManager.addEventlistener` and related code was removed because this line in particular was causing crashes in some cases when the app was installed and run on new devices through the Google Firebase or the Prelaunch Report mechanism.

Since sporks were not used for anything in the app, I removed this code on the master branch at https://github.com/dashevo/dash-wallet/commit/e237a37120d454c507614dab46c8a7055262997c (#334)